app/backend/app.py

```diff
@@ -186,6 +189,10 @@ async def setup_clients():
 
 
 def create_app():
+    if APPLICATIONINSIGHTS_CONNECTION_STRING:
+        configure_azure_monitor()
```

Comment:
does this default to 100% sampling rate or smaller? Is the default appropriate for production or should we make it obvious how to change the rate?

Reply:
Yes, 100%. It shouldn't interfere with performance because it uses batch span exporter (waits until idle to send events and is non-blocking for most things)